lib/loggery/metadata/middleware/shoryuken.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Shoryuken middleware that adds basic metadata to all log lines.
+
+module Loggery
+  module Metadata
+    module Middleware
+      class Shoryuken
+        include Loggery::Util
+
+        # Clients can provide their own error handler
+        class << self
+          attr_accessor(:error_handler) { ->(e) { Rails.logger.error(e) } }
```

Comment:
Maybe the comment should go to the next line?
```suggestion
        class << self
          # Clients can provide their own error handler
          attr_accessor(:error_handler) { ->(e) { Rails.logger.error(e) } }
```